### PR TITLE
update links to plotly's fork of d3-sankey-circular

### PR DESCRIPTION
**Pull request by archmoj**
@plotly/plotly_js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "d3"
   ],
   "bugs": {
-    "url": "https://github.com/tomshanley/d3-sankey-circular/issues"
+    "url": "https://github.com/plotly/d3-sankey-circular/issues"
```
Comment (antoinerg) [inline package.json]: It's probably fine for bugs to be reported upstream. Maybe we should keep the original URL here.

**Comment by antoinerg** [comment]
:woman_dancing:

**Comment by archmoj** [comment]
@antoinerg I don't have a write access to merge this PR.

**Comment by antoinerg** [comment]
> @antoinerg I don't have a write access to merge this PR.

It doesn't seem like I can grant you access. In the meantime, I went ahead and merged the current PR.

**Comment by archmoj** [comment]
@antoinerg are you going to publish a patch?

**Comment by alexcjohnson** [comment]
@archmoj @antoinerg for future use I've extended admin access for this repo to the @plotly/plotly_js team.

**Pull request merged**